lib/commands/general.js

```diff
@@ -254,6 +254,15 @@ commands.mobileUiautomatorPageSource = async function mobileUiautomatorPageSourc
   return await this.espresso.jwproxy.command(`/appium/execute_mobile/uiautomator_page_source`, 'GET');
 };
 
+
+/**
+ *  Change server context to jetpack-compose or espresso
+ */
+commands.mobileSetServerContext = async function mobileSetServerContext (opts = {}) {
```

Comment:
I would say we don't need this mobile command anymore as we could use settings

Reply:
removed